pyproject.toml

```diff
@@ -32,7 +32,7 @@ classifiers = [
 requires-python = ">= 3.11"
 
 dependencies = [
-    "snakemake >= 8",
+    "snakemake >= 8,<9.10.0",
```

Comment:
Why the cap ?

Reply:
This seems solved already.
https://github.com/snakemake/snakemake/releases/tag/v9.11.2